parachain/runtime/interlay/src/lib.rs

```diff
@@ -318,7 +319,10 @@ pub type SlowAdjustingFeeUpdate<R> =
     TargetedFeeAdjustment<R, TargetBlockFullness, AdjustmentVariable, MinimumMultiplier>;
 
 parameter_types! {
-    pub const TransactionByteFee: Balance = 0;  // TODO: this is 0 so that we can do runtime upgrade without fees. Restore value afterwards!
+    // TODO: this is 0 so that we can do runtime upgrade without fees. Restore value afterwards!
+    pub const TransactionByteFee: Balance = 0;
```

Comment:
Probably we should set this to something non-zero as well?

Reply:
We won't want fees initially to do the runtime upgrades as with Kintsugi.

Reply:
Right that makes sense